src/main/java/core/basesyntax/services/impl/ReportGeneratorServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.services.ReportGeneratorService;
+
+public class ReportGeneratorServiceImpl implements ReportGeneratorService {
```

Comment:
According to the checklist, we should not begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class implementation.